### PR TITLE
Adds Diagnostic HUD sunglasses

### PR DESCRIPTION
**Pull request by lilacrhododendron**
:cl: Tlaltecuhtli and Subject217
add: You can now make Diagnostic HUD sunglasses.
sprite: Fixed the MedHUD sunglasses having the wrong color when worn.
/:cl:

Now it actually merges. Tlal, I can teach you how to fast forward your master branch effortlessly without having to PR it.

### CODE REVIEW

**Review by ShizCalev — APPROVED** [review]

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -71,6 +71,14 @@
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
 	glass_colour_type = /datum/client_colour/glass_colour/green
 
+/obj/item/clothing/glasses/hud/diagnostic/sunglasses
+	name = "diagnostic sunglasses"
+	desc = "Sunglasses with a diagnostic HUD."
+	icon_state = "sunhuddiag"
+	item_state = "glasses"
+	flash_protect = 1
```
Comment (yoyobatty) [inline code/modules/clothing/glasses/hud.dm]: TRUE

**Review by yoyobatty — CHANGES_REQUESTED** [review]

File: code/modules/clothing/glasses/hud.dm (hunk 1 above)
Comment (subject217) [reply code/modules/clothing/glasses/hud.dm]: the whole file is that way, you're welcome to spend 15 minutes fixing it

Comment (yoyobatty) [reply code/modules/clothing/glasses/hud.dm]: It's your pr lad

Comment (GunHog) [reply code/modules/clothing/glasses/hud.dm]: `flash_protect ` is an integer, not a boolean. I have seen it go up to 2 and as low as -1.

Comment (subject217) [reply code/modules/clothing/glasses/hud.dm]: oh yeah, what was I thinking. yeah, flash_protect has multiple settings, light flash protection and heavy flash protection

**Comment by yoyobatty** [comment]
Ah crud oops

**Comment by ExcessiveUseOfCobblestone** [comment]
mmmm `flash_protect` and `tint` doesn't have defines.

**Pull request merged**